extensions/playnite-launcher/README.md

```diff
@@ -0,0 +1,12 @@
+# Playnite Launcher
+
+Launch your favorite games from Raycast using Playnite.
+
+## Requirements
+
+1. [Playnite](https://playnite.link/)
+2. [FlowLauncherExporter](https://github.com/Garulf/FlowLauncherExporter/releases/latest)
```

Comment:
What are users to do with the GitHub link? Might be worth enhancing the README here.